### ISSUE TITLE
Installed models on Civitai show as installed on Civitai even if only one version installed. 

### ISSUE THREAD

**Issue by Brennok** [civitai/civitai-link-desktop#27]
Not sure the best way to fix this so figured I would throw it out there.  This may be better directed to site feedback so let me know.

Currently installed will show if a model is installed when browsing models. If the model has multiple version IDs, it will show installed even if only one modelversionID is installed. 

Example https://civitai.com/models/351631 has 3 version IDs and I have 2 of the 3, it still shows installed on the models page though. 
![Screenshot 2024-04-15 161753 - models view](https://github.com/civitai/civitai-link-desktop/assets/52268872/9bdb928e-48b9-4444-a155-3d2d0b1ded89)
![Screenshot 2024-04-15 161833 not installed](https://github.com/civitai/civitai-link-desktop/assets/52268872/2f6ea141-249a-4280-b109-ed9fb1defb9d)
![Screenshot 2024-04-15 161859 - installed](https://github.com/civitai/civitai-link-desktop/assets/52268872/864d503c-283c-4cda-a14a-8adaa9a7f2e3)

If possibly it would be great, if there was another icon that showed if you only had some of the models even if just a different color checkmark like a yellow checkmark instead of green.

This is more problematic when you have some of the style variants like the Pony Style where multiple different styles are under one model number. 
https://civitai.com/models/264290/styles-for-pony-diffusion-v6-xl-not-artists-styles

**Comment by cmoody** [comment]
@Brennok yeah feedback through here or office hours works. Im checking with the team to see how we might want to handle this. Ill put it in the task list.
